### ISSUE TITLE
Rearrange menu items and improve legibility

### ISSUE THREAD

**Issue by Aashishkebab** [rsanchez-wsu/RaiderPlanner#45]
Remove the hamburger menu for navigation and instead implement them as "tabs" at the bottom of the screen. This prevents the need to wait for the slow and choppy animation of the menu every time we need to change pages.

**Comment by Aashishkebab** [comment]
I would like to work on this issue if it is approved.

**Comment by rsanchez-wsu** [comment]
@Aashishkebab I am not convinced that this is a good idea.  The hamburger menu has become somewhat standard as far as UI goes and if we deploy on mobile (as described in #26) or web (as described in #27) then I am not sure that the tabs would work as well.  This issue can remain open for the time being, but I suggest that we wait on doing anything with it.

**Comment by Aashishkebab** [comment]
Here is a rough overview of what it kind've would look like. The "calendar" button
<img width="742" alt="screenshot_20170922-155002" src="https://user-images.githubusercontent.com/25588487/30761935-139513e4-9fae-11e7-8691-00a83d485205.png">
 needs to be changed into an icon if possible. And we can add an "Add Milestone" button to the bottom right Material Design style.

**Comment by rsanchez-wsu** [comment]
@Aashishkebab That looks good.  I think I would like to make sure that we think about what other buttons/icons/etc. may belong in the toolbar and then once we have that set we can begin to make changes.  I would also like input from other folks on the project so it is not just the opinions of one or two people.

**Comment by Aashishkebab** [comment]
Here is what the finished product will mostly look like. I've made changes to the colors to improve legibility, and I've also added ToolTips (not seen in screenshot). Finally, items have been moved around. I have already issued a Pull Request for this, but I thought I'd post it here as well.

<img width="751" alt="screenshot_20170923-005147" src="https://user-images.githubusercontent.com/25588487/30770255-8dbf878a-9ff9-11e7-8bf2-9845be88bad1.png">
